scripts/build-images

```diff
@@ -83,6 +83,7 @@ xargs -n1 -t docker image pull --quiet << EOF > build/images-multus.txt
     ${REGISTRY}/rancher/hardened-sriov-network-resources-injector:v1.5-build20231009
     ${REGISTRY}/rancher/hardened-sriov-network-webhook:v1.2.0-build20230607
     ${REGISTRY}/rancher/hardened-whereabouts:v0.6.2-build20231009
+    ${REGISTRY}/library/busybox:latest
```

Comment:
We can't use the `latest` tag, there are different semantics for pulling when that is used, and it is not suitable for airgap use. We need to use a fixed tag. It sounds like this might require another change to the chart though?

Reply:
I updated the chart to use the mirrored busybox from rancher